spring-data-jdbc/src/test/resources/org.springframework.data.jdbc.repository/JdbcRepositoryWithListsIntegrationTests-db2.sql

```diff
@@ -3,3 +3,20 @@ DROP TABLE dummy_entity;
 
 CREATE TABLE dummy_entity ( id BIGINT GENERATED BY DEFAULT AS IDENTITY ( START WITH 1 ) PRIMARY KEY, NAME VARCHAR(100));
 CREATE TABLE element (id BIGINT GENERATED BY DEFAULT AS IDENTITY (START WITH 1) PRIMARY KEY, content VARCHAR(100), Dummy_Entity_key BIGINT, dummy_entity BIGINT);
+
+CREATE TABLE root
```

Comment:
We need a drop table statement for each new table. This applies to the other databases as well. 

The test fails locally, probably when run multiple times, because the database gets reused and the create table fails when a database is already present.